Student.js

```diff
@@ -188,6 +188,12 @@ function std() {
 }
 
 function res() {
+for(k=0;k<100;k++){

```

Comment:
Initialize the loop variable 'k' with 'let' or 'const' to avoid potential scope leakage. [important]